handlers/whatsapp/whatsapp.go

```diff
@@ -839,6 +843,16 @@ func sendWhatsAppMsg(msg courier.Msg, sendPath *url.URL, payload interface{}) (s
 	req, _ := http.NewRequest(http.MethodPost, sendPath.String(), bytes.NewReader(jsonBody))
 	req.Header = buildWhatsAppHeaders(msg.Channel())
 	rr, err := utils.MakeHTTPRequest(req)
+
+	if rr.StatusCode == 429 || rr.StatusCode == 503 {
+		rateLimitKey := fmt.Sprintf("rate_limit:%s", msg.Channel().UUID().String())
+		rc.Do("set", rateLimitKey, "engaged")
+		rc.Do("expire", rateLimitKey, 2)
```

Comment:
doesn't the 429 response tell us when we can retry ?

Reply:
No, we get 

```
{"meta":{"api_status":"stable","version":"2.35.4"},"errors":[{"code":1015,"title":"Too many requests","details":"Rate limiting engaged - Too Many Requests."}]}
```

Reply:
so we're backing down for 2 seconds but do we have any information if that's a reasonable back down?

Reply:
We are pausing the queue to send for 2 seconds and try again once we have the 429 status we pause for another 2 seconds

Reply:
https://developers.facebook.com/docs/whatsapp/api/rate-limits#capacity

The rate limit we are having is that we had more than 50 requests per second so stopping 2 seconds seems that will help reset the limit

Reply:
ok can you add a little comment to the code here explaining the rational for 2 and maybe mention that in future it should use header values

Reply:
and then let's get this out there!